test/e2e/metamask.spec.js

```diff
@@ -581,67 +589,18 @@ describe('Metamask popup page', async function () {
   })
 
   describe('Add Token: Custom', function () {
-    const symbol = 'TST'
-    const decimals = '0'
 
-    describe('Token Factory', function () {
+   describe('Add token to LOCALHOST', function () {
 
-      it('navigates to token factory', async function () {
+      it('Create custom token in LOCALHOST', async function () {
         await setProvider(NETWORKS.LOCALHOST)
-        await driver.get('http://thetokenfactory.com/#/factory')
+        await createToken(account1, token, false)// skip address 0xa6... since POA core accidentally contains token with the same address
```

Comment:
I don't understand this. Why is it necessary to skip these addresses?

Reply:
fixed here https://github.com/poanetwork/nifty-wallet/pull/211